charts/backstage/Chart.yaml

```
@@ -26,22 +27,24 @@ dependencies:
 keywords:
   - backstage
   - idp
+  # remove reference to janus-idp when sunsetted https://issues.redhat.com/browse/RHIDP-3227
   - janus-idp
-kubeVersion: ">= 1.25.0-0"
+  - developer-hub
+  - redhat-developer-hub
+  - redhat
+kubeVersion: ">= 1.19.0-0"
 maintainers:
-  - name: Red Hat Developer Hub Team
-    url: https://github.com/redhat-developer/rhdh-chart
+  - name: Red Hat
+    url: https://redhat.com
 name: backstage
 type: application
-sources:
-  - https://github.com/redhat-developer/rhdh-chart
-  - https://github.com/redhat-developer/rhdh
+sources: []
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 4.0.0
+version: 4.0.1
```

```suggestion
version: 4.1.0
```

I think a minor bump would make more sense here.